api/src/cli/commands/schema/apply.ts

```diff
@@ -109,9 +113,18 @@ export async function apply(snapshotPath: string, options?: { yes: boolean }): P
 							}
 						}
 					} else if (diff[0]?.kind === 'D') {
-						message += `\n  - ${chalk.red('Delete')} ${collection}.${field} -> ${related_collection}`;
+						message += `\n  - ${chalk.red('Delete')} ${collection}.${field}`;
 					} else if (diff[0]?.kind === 'N') {
-						message += `\n  - ${chalk.green('Create')} ${collection}.${field} -> ${related_collection}`;
+						message += `\n  - ${chalk.green('Create')} ${collection}.${field}`;
+					} else if (diff[0]?.kind === 'A') {
+						message += `\n  - ${chalk.blue('Update')} ${collection}.${field}`;
+					} else {
+						continue;
+					}
+
+					// Related collection doesn't exist for m2a relationship types
+					if (related_collection) {
+						message += `-> ${related_collection}`;
```

Comment:
@rijkvanzanten Small nitpick: I think you're missing a space here right before the arrow.